functions/api/v1/accounts/[id].ts

```diff
@@ -21,41 +16,11 @@ export const onRequest: PagesFunction<Env, any, ContextData> = async ({ request,
 }
 
 export async function handleRequest(domain: string, id: string, db: D1Database): Promise<Response> {
-	const handle = parseHandle(id)
+	const account = await getAccount(domain, id, db)
 
-	if (handle.domain === null || (handle.domain !== null && handle.domain === domain)) {
-		// Retrieve the statuses from a local user
-		return getLocalAccount(domain, db, handle)
-	} else if (handle.domain !== null) {
-		// Retrieve the statuses of a remote actor
-		const acct = `${handle.localPart}@${handle.domain}`
-		return getRemoteAccount(handle, acct)
+	if (account) {
+		return new Response(JSON.stringify(account), { headers })
 	} else {
-		return new Response('', { status: 403 })
```

Comment:
@xtuc you can notice that the 403 is gone in my code, I think that it was never anyway returned, because if `handle.domain` was `null` then you'd go in the first branch, if it was different from `null` you'd either go in the first branch or the second, so I don't think the last else was actually doing anything

please let me know if I'm wrong 🙂

Reply:
Yeah that's correct. The response was mostly for typing here. Thanks for checking